website/blog/2025-05-15-podman-desktop-core.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Containers and Kubernetes development with Podman Desktop
+description: A step-by-step guide for your containers and Kubernetes development with Podman Desktop
+authors: [firewall]
+tags: [podman-desktop, podman, development, container, tools]
+hide_table_of_contents: false
+---
+
+In the world of modern software development, containers and Kubernetes are no longer optional; they are essential. That’s where Podman Desktop comes in, your ultimate tool for building, managing, and deploying containers and Kubernetes clusters with ease and confidence. In this blogpost we will walk through a typical development workflow, creating and building a container and then testing it in a local Kubernetes cluster. Let’s dive in.
+
+## Building Containerized Applications with Podman Desktop
+
+First, let's start by building an application in a container with Podman Desktop.We need our application code and a Containerfile. For a simple application, your Containerfile might look something like:
+
+```Dockerfile
+FROM nginx
+
+COPY ./index.html /usr/share/nginx/html/
```

Comment:
I don't see the content of the index.html 

so if I want to copy/paste the Containerfile will break

Reply:
Added inline